src/AfterLogin/CardComponent/EditCardForm.js

```diff
@@ -0,0 +1,247 @@
+import { useMemo, useState } from "react";
+import Translator from "../../Controller/Translator";
+import { Autocomplete, Button, Card, TextField } from "@mui/material";
+import ErrorPage from "../../ErrorPage";
+import LoadingButton from "@mui/lab/LoadingButton";
+import TextToSpeech from "../../Controller/TextToSpeech";
+import VolumeUpIcon from "@mui/icons-material/VolumeUp";
+import EditIcon from "@mui/icons-material/Edit";
+import CheckIcon from "@mui/icons-material/Check";
+import ClearIcon from "@mui/icons-material/Clear";
+import "./EditCardForm.css";
+import { useTheme } from "@mui/material/styles";
+
+export default function EditCardForm(props) {
```

Comment:
```suggestion
export default function EditCardForm({ card, editing, ...props}) {
```

This way you do not need to access `props` everytime you want to lookup the editing key